src/utils/maak_protocol.R

```diff
@@ -0,0 +1,263 @@
+                                 generiek = "0",
+                                 water = "1",
+                                 bodem = "2",
+                                 lucht = "3",
+                                 vegetatie = "4",
+                                 soorten = "5"
+  )
+  
+  alle_nummers <- geef_protocolnummers(protocol_type = protocol_type)
+  
+  if (length(alle_nummers) == 0) {
+    protocol_volgnummer <- "01"
+  } else {
+    nummers <- alle_nummers[
+      str_detect(alle_nummers, 
+                 pattern = paste0("^", protocol_hoofdnummer)
+      )]
+    protocol_volgnummer <- max(
+      as.integer(
+        str_extract(nummers, "\\d{2}$"))
+    ) + 1
+    protocol_volgnummer <- formatC(protocol_volgnummer, 
+                                   width = 2, format = "d", flag = "0")
+  }
+  
+  protocol_nummer <- paste0(protocol_hoofdnummer,
+                            protocol_volgnummer)
+  
+  korte_titel <- tolower(korte_titel)
+  korte_titel <- str_replace_all(korte_titel, " ", "-")
+  korte_titels <- geef_korte_titels(protocol_type = protocol_type)
+  assert_that(!(korte_titel %in% korte_titels), 
+              msg = "De opgegeven korte titel bestaat al. Geef een korte titel die nog niet bestaat.")
+  
+  protocol_code <- paste0(protocol_type, "-", protocol_nummer)
+  folder_name <- paste0(protocol_code, "_", korte_titel)
+  folder_name <- tolower(folder_name)
+  
+  # directory setup
+  if (missing(projectnaam)) {
+    path_to_protocol <- file.path(project_root, "src", "thematic", thematiek, 
+                                  folder_name)
+  } else {
+    path_to_protocol <- file.path(project_root, "src", "project", projectnaam, 
+                                  thematiek, folder_name)
+  }
+  
+  # check for existence of the folder
+  if (dir.exists(path_to_protocol)) {
+    stop(sprintf(paste0("De protocol repository bevat reeds ",
+                        "een map voor %s!"), path_to_protocol))
+  } else {
+    # create a new directory
+    dir.create(file.path(path_to_protocol), 
+               recursive = TRUE)
+  }
+  
+  # move all files from the template folder and add info
+  files <- list.files(path = path_to_template, 
+                      full.names = FALSE,
+                      recursive = FALSE, 
+                      pattern = protocol_type)
+  
+  for (filename in files) {
+    print(paste0("Preparing: ", filename))
+    new_file_path <- file.path(path_to_protocol, filename)
+    original_file_path <- file.path(path_to_template, filename)
+   
+    # create new file
+    file.create(new_file_path)
+    file.rename(from = new_file_path, 
+                to = file.path(path_to_protocol, paste0(folder_name, ".Rmd")))
+    new_file_path <- file.path(path_to_protocol, paste0(folder_name, ".Rmd"))
+    
+    # add original file content from reference, changing values
+    original_file <- file(original_file_path, "r")
+    original_file_content <- readLines(original_file)
```

Comment:
```suggestion
    original_file_content <- readLines(original_file_path)
```
this should work too. And makes L148 and L163 obsolete.